### PR TITLE
Rearranged bootstrap sass to remain coherent

### PR DESCRIPTION
**Pull request by michaelbeil**
with L5 introduction.

Let me know if you want this PR against a different branch.

:coffee: :package:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ For more installation notes, refer to the [Install gulp and Bower](#install-gulp
 * [BrowserSync](http://www.browsersync.io/) for keeping multiple browsers and devices synchronized while testing, along with injecting updated CSS and JS into your browser while you're developing
 * [Bower](http://bower.io/) for front-end package management
 * [asset-builder](https://github.com/austinpray/asset-builder) for the JSON file based asset pipeline
-* [Sass](https://github.com/twbs/bootstrap-sass) [Bootstrap](http://getbootstrap.com/)
+* [Bootstrap](http://getbootstrap.com/)[Sass](https://github.com/twbs/bootstrap-sass)
```
Comment (QWp6t) [inline README.md]: There should be a space in there.

**Comment by retlehs** [comment]
this was done on purpose. mdo mentioned on bootstrap slack that this makes more sense. i originally had it backwards.

**Comment by QWp6t** [comment]
I personally would use:

[Bootstrap](http://getbootstrap.com/) for [Sass](https://github.com/twbs/bootstrap-sass)

Since that's how they title it in their README.md

**Comment by retlehs** [comment]
i also had that. lol.

**Comment by austinpray** [comment]
"Sass [version of] Bootstrap" makes more sense than "Bootstrap [???] Sass". 

Race horse 
Mineral water 
Book case

**Comment by austinpray** [comment]
Lol you guys beat me too it

**Comment by michaelbeil** [comment]
Just include all of the above.

File: README.md (hunk 1 above)
Comment (michaelbeil) [inline README.md]: yea, totes forgot my spacing, but at any rate this has been closed out :sweat_smile:.

**Comment by michaelbeil** [comment]
@retlehs, would it make sense to change the intro to Sass Bootstrap to match the semantics here? Or is that a, "nah, it's all semantics anyways."? :tada:

**Comment by austinpray** [comment]
"Bootstrap Sass" is about to be the default anyway. BS4 will be Sass by default 😎

**Comment by retlehs** [comment]
yeah that needs to be removed or changed in the intro

**Pull request closed without merging**